src/EPR.Calculator.Service.Function/Services/CalculatorRunService.cs

```diff
@@ -1,56 +1,161 @@
+        /// <param name="pipelineSucceeded">Indicates whether the pipeline succeeded.</param>
+        /// <returns>The JSON content for the status update.</returns>
+        public static StringContent GetStatusUpdateMessage(int calculatorRunId, bool pipelineSucceeded)
+        {
+            var statusUpdate = new
+            {
+                runId = calculatorRunId,
+                updatedBy = "string",
+                isSuccessful = pipelineSucceeded,
+            };
+
+            return new StringContent(
+                JsonSerializer.Serialize(statusUpdate),
+                Encoding.UTF8,
+                "application/json");
+        }
+
+        /// <summary>
+        /// Creates a JSON message containing the calculator run ID for preparing calculation results.
+        /// </summary>
+        /// <param name="calculatorRunId">The ID of the calculator run.</param>
+        /// <returns>A <see cref="StringContent"/> object containing the JSON message.</returns>
+        public static StringContent GetPrepareCalcResultMessage(int calculatorRunId)
+        {
+            var calcResultsRequest = new
+            {
+                runId = calculatorRunId,
+            };
+
+            return new StringContent(
+                JsonSerializer.Serialize(calcResultsRequest),
+                Encoding.UTF8,
+                "application/json");
+        }
+
+        /// <summary>
+        /// Starts the calculator process.
+        /// </summary>
+        /// <param name="calculatorRunParameter">The parameters required to run the calculator.</param>
+        /// <returns>
+        /// A task that represents the asynchronous operation. The task result contains a boolean indicating success or failure.
+        /// </returns>
+        public async Task<bool> StartProcess(CalculatorRunParameter calculatorRunParameter)
         {
             this.logger.LogInformation("Process started");
+            bool isPomSuccessful = false;
+            bool runRpdPipeline = bool.Parse(Configuration.ExecuteRPDPipeline);
+            var response = new HttpResponseMessage(HttpStatusCode.Continue);
+            bool isSuccess = response.IsSuccessStatusCode;
+
+            if (runRpdPipeline)
+            {
+                var orgPipelineConfiguration = GetAzureSynapseConfiguration(
+                    calculatorRunParameter,
+                    Configuration.OrgDataPipelineName);
 
-            var pomPipelineConfiguration = GetAzureSynapseConfiguration(
-                calculatorRunParameter,
-                Configuration.PomDataPipelineName);
-            var isPomSuccessful = await azureSynapseRunner.Process(pomPipelineConfiguration);
+                var isOrgSuccessful = await this.azureSynapseRunner.Process(orgPipelineConfiguration);
 
-            this.logger.LogInformation("Pom status", isPomSuccessful);
+                this.logger.LogInformation("Org status: {Status}", Convert.ToString(isOrgSuccessful));
```

Comment:
Do we need to Convert.ToString(isOrgSuccessful)
Is it not string by default?

Reply:
Resolved in PR#16